content/guides/component-testing/mounting-vue.md

```diff
@@ -80,6 +80,21 @@ describe('<Stepper>', () => {
 
 Here, we have a single test that ensures that our component mounts.
 
+If you already have a component that you want to create a test for and you don't
```

Comment:
If they are following this guide, they already have the stepper component to test so the wording here might be confusing. I think we should probably just show how to use the app to create a spec and get rid of the section directly above on creating the spec file manually. They might think they need to do both here.